arxivdigest/api/README.md

```diff
@@ -234,6 +234,15 @@ Fields returned for each user:
   - `feedback`: is the feedback stored in a dictionary of {feedback_type: datetime}
     -  "seen":   datetime of when article was seen or null if not seen
     -  "clicked":     datetime of when topic was clicked or null if not clicked
+    -  "state": USER_ADDED, USER_REJECTED, EXPIRED, REFRESHED, SYSTEM_RECOMMENDED_ACCEPTED, SYSTEM_RECOMMENDED_REJECTED
+
+Explanations of different states:
+  - USER_ADDED: The user added the topic themselves by writing it.
+  - USER_REJECTED: The user removed a topic from their profile that was previously USER_ADDED.
+  - EXPIRED: The topic was recommended, but the user did not interact with it within 24 hours.
+  - REFRESHED: The topic was recommended, but the user refreshed the topic suggestion list without interacting with it.
+  - SYSTEM_RECOMMENDED_ACCEPTED: The topic was recommended and the user accepted it from the list or manually added it after rejecting it.
```

Comment:
If manually added after rejecting a system recommendation, then the state should be USER_ADDED.